retrieval/target_test.go

```diff
@@ -293,7 +293,7 @@ func TestTargetScrapeTimeout(t *testing.T) {
 	)
 	defer server.Close()
 
-	testTarget := newTestTarget(server.URL, 25*time.Millisecond, clientmodel.LabelSet{})
```

Comment:
The reported issue was that a request timed out that shouldn't have. I.e. when execution reached the request the 25ms have already passed. So we don't want to hit the deadline.

For the case where we want to hit the timeout even the 5ms are unnecessary.